### ISSUE TITLE
Visualisation error with zoomifytilesource.js

### ISSUE THREAD

**Issue by arrigo55** [openseadragon/openseadragon#1169]
I tested the zoomifytilesource with two different Zoomify images. 
The first one https://drive.google.com/open?id=0B1mkW7rsGhsPWEN6M3gxZi1ScFk is small (width: 4096 height: 4096) and the visualisation with Openseadragon is perfect. 
The second one https://drive.google.com/open?id=0B1mkW7rsGhsPR2IzR0x3OHM2QmM is a large image (width: 81920 height: 103349) and the visualisation is poor.
The images can be seen at
http://www.syliconstuga.cloud/osd-zmf/ops/small   = small one visualised with with Openseadragon
http://www.syliconstuga.cloud/osd-zmf/zmf/small   = small one visualised with Zoomify

http://www.syliconstuga.cloud/osd-zmf/ops/large   = large one visualised with with Openseadragon
http://www.syliconstuga.cloud/osd-zmf/zmf/large   = large one visualised with with Zoomify

The ImageProperties.xml files are included in the two downloads. 
Any suggestion? Thanks

**Comment by iangilman** [comment]
@foobarable would you mind taking a look at this?

**Comment by foobarable** [comment]
@iangilman Yes, I can do this, but it will need some time.

**Comment by foobarable** [comment]
@arrigo55 How did you create the zoomify images?

**Comment by foobarable** [comment]
@arrigo55 You did not specify the height of the image correctly: Compare the height in your javascript with the height of the ImageProperties.xml.

**Comment by foobarable** [comment]
@iangilman Probably we still should add the functionality to read the dimensions from the ImageProperties.xml file.

**Comment by arrigo55** [comment]
Yes, there was a mistake in the Height parameter of my index.html file. Now it is correct, but the result is still quite different from the espected one.

**Comment by iangilman** [comment]
@foobarable Yes, being able to get the dimensions from the ImageProperties.xml would be good! That's how it works with DZI, for instance. Keep in mind that the image dimensions are different than the viewport coordinates for the image. That said, we always force the viewport coordinates to have the same aspect ratio as the image, even if the scale is different.

**Comment by foobarable** [comment]
@arrigo55 In your example it looks good now. What did you change?

**Comment by foobarable** [comment]
It seems there is some error after zoomlevel 6 so the tiles are loaded from the wrong tilegroup

**Comment by iangilman** [comment]
Interesting... glad you're making progress on tracking down the issue! :)

**Comment by arrigo55** [comment]
I simply put the correct height value in the javascript. The result looks
good only at low magnification. As you said in one of the following posts
it seems there is something wrong at high magnification. Thank you for your
precious work.

2017-04-24 8:28 GMT+02:00 foobarable <notifications@github.com>:

**Comment by foobarable** [comment]
I currently don't have the time to analize the issue here, so the solution may be delayed. Sorry

**Comment by iangilman** [comment]
Okay, thanks for the update. Hopefully you can get to it eventually!

If anyone else wants to take a look, please do :)
